src/components/BoardColumn/ColumnHeader.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-  <div class="flex justify-between pb-4">
+  <div
+    class="bg-gradient-to-r from-teal-400 to-blue-500 focus:from-pink-500 focus:to-orange-500 text-white font-semibold px-6 py-3 rounded-tr-lg rounded-tl-lg flex justify-between pb-4"
+  >
```

Comment:
Can we use here the new color palette? for buttons we´re using variations of indigo-800 or rose-800 and for nav-bar/footer, gray-500